doc/python_api/rst/bge_types/bge.types.KX_WorldInfo.rst

```diff
@@ -102,3 +102,16 @@ base class --- :class:`PyObjectPlus`
 
       :type: float from 0.0 to infinite
 
+   .. attribute:: envLightEnable
```

Comment:
There is already an RW api for "mistEnable".
I thought there was enough RW and as Maujoe asked for RO, I left in RO.

About the name, I already said that I prefered mistEnabled, envLightEnabled, but it has been decided that it will be enable without "d",  so I did like that.